sdk-python/livescoreapi.py

```diff
@@ -6,18 +6,69 @@
 @author: shrey.aryan
 """
 
+from urllib.parse import urlencode
 import urllib.request
-
+import json
 
 class livescores:
+    """
+    A class used to get livescores. 
+    ...
+
+    Attributes
+    ----------
+    key : str
+        an alphanumeric string used to access data from the LiveScore API
+    secret : str
+        an alphanumeric string used to access data from the LiveScore API
+    
+    Methods
+    -------
+    _get_html_file(link)
     
-    def get_all(key,secret):
-        html_file = urllib.request.urlopen("http://livescore-api.com/api-client/scores/live.json?key=" + key + "&secret=" + secret)
-        json_data = html_file.read()
-        return json_data
+    get_all()
     
+    """
+    parent_link = "http://livescore-api.com/api-client"
 
+   
+    def __init__(self, key, secret):
+        key.strip()
```

Comment:
This will not do much as it will strip, but it will not affect the key. It should be 
`key = key.strip()` the same applies for the `secret`.